.github/workflows/wheels.yml

```diff
@@ -99,14 +99,26 @@ jobs:
         - [macos-14, macosx_arm64]
         - [windows-2022, win_amd64]
         # TODO: support PyPy?
-        python: [["cp310", "3.10"], ["cp311", "3.11"], ["cp312", "3.12"]]
-
+        python: [["cp310", "3.10"], ["cp311", "3.11"], ["cp312", "3.12"], ["cp313", "3.13"], ["cp313t", "3.13"]]
+        include:
+        # 'pip' is the default
+        - cibw_build_frontend: pip

```

Comment:
Can you put this in the `CIBW_BUILD_FRONTEND` variable declaration i.e. `CIBW_BUILD_FRONTEND: ${{ matrix.cibw_build_frontend || 'pip '}}